### PR TITLE
iTIP Reschedule Fix

### PR DESCRIPTION
**Pull request by ksmurchison**
Per RFC 5545, Section 4.4.7:  remove overrides if not included with reschedule of master

### CODE REVIEW

File: cassandane/Cassandane/Cyrus/Sieve.pm [hunk 1]
```diff
@@ -4371,6 +4371,138 @@ EOF
...
+EOF
+
+    xlog $self, "Deliver iMIP update";
+    $msg = Cassandane::Message->new(raw => $imip);
+    $msg->set_attribute(uid => 2,
+                        flags => [ '\\Recent', '\\Flagged' ]);
+    $self->{instance}->deliver($msg);
+
+    xlog $self, "Check that the message made it to INBOX";
+    $self->check_messages({ 1 => $msg }, check_guid => 0);
```
Comment (rsto) [inline cassandane/Cassandane/Cyrus/Sieve.pm:4494]: Is that right to check `1` here?

**Review by rsto — COMMENTED** [review]
I have a hard time figuring out the logic is sane, and I think that's just due to the variable names and (probably outdated) comments.

File: cassandane/Cassandane/Cyrus/Sieve.pm (hunk 1 above)
Comment (ksmurchison) [reply cassandane/Cassandane/Cyrus/Sieve.pm]: This check isn't required for this test so I removed it.

**Review by rsto — APPROVED** [review]
Clearer to me now. LGTM

**Pull request merged**